.github/CHANGELOG.md

```diff
@@ -343,8 +336,58 @@
   res = qml.grad(cost_fn)(params)
   ```
 
+  <h4>PennyLane now comes packaged with <code>lightning.qubit</code></h4>
```

Comment:
@antalszava since this is a very big feature, I would add this either above or below the pytorch entry (maybe below? --- your call :) )

Reply:
Moved it above, thanks!